pages/cloudflare/migrate-from-0.2.mdx

```diff
@@ -80,4 +80,9 @@ The name of the CLI was changed to `opennextjs-cloudflare`:
 ##### 5 Add `.open-next` to `.gitignore`
 
 You should change `.worker-next` to `.open-next` in your `.gitignore` file to prevent the build output from being committed to your repository.
-You can safely delete the content of the now unused `.worker-next`.
\ No newline at end of file
+You can safely delete the content of the now unused `.worker-next`.
+
+##### 6. Apply 0.3.x updates
```

Comment:
Or do we want to rename the section "migrate from 0.2 to 0.3" ?
(i.e. we don't wank to cascade every minor updates)

Reply:
Sure, sounds better to me 👍

Reply:
done https://github.com/opennextjs/docs/pull/57/commits/43fb44e2ec374b9dc2b0e3d356ae470fb2f6b460